src/pyscaffold/templates/tox_ini.template

```diff
@@ -8,6 +8,8 @@ envlist = py27,py34,py35,py36,flake8
 skip_missing_interpreters = True
 
 [testenv]
+# uncomment `usedevelop` for faster tests without package builds & installs
```

Comment:
How about adding:
```
...
# but please be aware that you will not be testing the package as if it was installed by the final user
# see http://tox.readthedocs.io/en/latest/example/general.html#avoiding-expensive-sdist
```

Or something like that?

Reply:
I would like to avoid two line comments and too many deep links into the documentation of others that get outdated quite easily. The main tox documentation is already in the head of the file addressed. What do you think about the following change below?

Reply:
:+1: 
Very clear, it should be enough!
Thanks